src/Symfony/Component/Intl/Currencies.php

```diff
@@ -139,6 +139,156 @@ public static function forNumericCode(int $numericCode): array
         return self::readEntry(['NumericToAlpha3', (string) $numericCode], 'meta');
     }
 
+    /**
+     * @param non-empty-string $country e.g. 'FR'
+     * @param ?bool $legalTender if the currency must be a legal tender. Using null do not filter anything
+     * @param bool $activeOnly exclude currencies that are not active on the given $date in countries
+     * @param string $date The date string on which the check will be performed
+     *
+     * @throws MissingResourceException if the given $country does not exist
+     *
+     * @return array<non-empty-string, array{from?: non-empty-string, to?: non-empty-string, tender?: bool}>
```

Comment:
im not sure this is the proper return type compared to `getCurrencyCodes` and `forNumeric`, unexpected at least i think